src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_adexchangebuyer.v1.4.json.baseline

```diff
@@ -146,13 +149,13 @@ namespace AdExchangeBuyerSample
             int id = 0;
 
             // TODO Add code to assign values to fields of 'content'.
```

Comment:
Nit - properties instead of fields? (Can be done later; it's not a change in this PR anyway.)